x-pack/plugins/reporting/server/config/schema.test.ts

```diff
@@ -62,4 +62,27 @@ describe('Reporting Config Schema', () => {
       );
     }
   );
+  it('disables image reporting with serverless', () => {
+    expect(() =>
+      ConfigSchema.validate({ export_types: { pdf: { enabled: true } } }, { serverless: true })
+    ).toThrow();
+    expect(() =>
+      ConfigSchema.validate({ export_types: { png: { enabled: true } } }, { serverless: true })
+    ).toThrow();
+    // csv should pass in either dev or serverless
+    expect(() =>
+      ConfigSchema.validate({ export_types: { csv: { enabled: true } } }, { serverless: true })
+    ).not.toThrow();
+  });
+  it('allows image reporting in dev', () => {
```

Comment:
it should allow image reporting for any non-serverless config

Reply:
I like this phrasing much better thanks!